src/brogue/Recordings.c

```diff
@@ -674,12 +674,19 @@ void advanceToLocation(unsigned long destinationFrame) {
 
     while (rogue.playerTurnNumber < destinationFrame && !rogue.gameHasEnded && !rogue.playbackOOS) {
         if (useProgressBar && !(rogue.playerTurnNumber % progressBarInterval)) {
-            rogue.playbackFastForward = false;
+            rogue.playbackFastForward = false; // so that pauseBrogue looks for inputs
             printProgressBar((COLS - 20) / 2, ROWS / 2, "[     Loading...   ]",
                              rogue.playerTurnNumber - initialFrameNumber,
                              destinationFrame - initialFrameNumber, &darkPurple, false);
+            while (pauseBrogue(0)) { // pauseBrogue(0) is necessary to flush the display to the window in SDL
+                if (rogue.gameHasEnded) {
+                    return;
+                }
+                rogue.playbackFastForward = true; // prevent monster flashes from showing up on screen
+                nextBrogueEvent(&theEvent, true, false, true); // eat the input if it isn't clicking x
+                rogue.playbackFastForward = false;
```

Comment:
So I noticed you added a condition in nextBrogueEvent for this variable, couldn't we set just `rogue.creaturesWillFlashThisTurn = false` before the call?

Reply:
That definitely works, not sure what I was thinking lol. First I added a fifth parameter to nextbrogueevent, next I did what I did here... not sure how I missed this.